2023-02-17 02:46:49: WARNING: connection.py:611 * 140334579431168 Connection dropped: socket connection error: Connection refused
2023-02-17 03:24:41: WARNING: connection.py:611 * 140334579431168 Connection dropped: socket connection error: Connection refused
2023-02-17 04:35:56: WARNING: connection.py:611 * 140334579431168 Connection dropped: socket connection error: Connection refused
2023-02-17 05:20:39: WARNING: connection.py:611 * 140334579431168 Connection dropped: socket connection error: Connection refused
2023-02-17 06:26:26: WARNING: connection.py:611 * 140334579431168 Connection dropped: socket connection error: Connection refused
2023-02-17 07:07:01: WARNING: connection.py:611 * 140334579431168 Connection dropped: socket connection error: Connection refused
2023-02-18 02:16:16: WARNING: connection.py:611 * 139933918422784 Connection dropped: socket connection broken
2023-02-18 02:16:16: WARNING: connection.py:615 * 139933918422784 Transition to CONNECTING
2023-02-18 02:16:16: ERROR: zk_manager.py:88 * 139933918422784 connecting to zookeeper failed! will restart
2023-02-18 02:16:17: WARNING: connection.py:624 * 139933918422784 Session has expired
2023-02-18 02:16:17: ERROR: zk_manager.py:88 * 139933918422784 connecting to zookeeper failed! will restart
